tests/__init__.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+import tempfile
+from collections.abc import Sequence
+from typing import Optional
+from types import TracebackType
+
+from mkosi.distributions import Distribution, detect_distribution
+from mkosi.log import die
+from mkosi.run import run
+from mkosi.types import CompletedProcess
+from mkosi.util import INVOKING_USER
+
+
+class Image:
+    def __init__(self, options: Sequence[str] = []) -> None:
+        self.options = options
+
+        if d := os.getenv("MKOSI_TEST_DISTRIBUTION"):
+            self.distribution = Distribution[d]
```

Comment:
How about passing the distribution and release as arguments here, this would allow to mark them, e.g. `pytest.mark.arch` and `pytest.mark.fedora` and then the tests can be parametrized over this and called via the marks.

Reply:
I'm not sure I understand. Maybe do a follow up PR after this one is merged?

Reply:
It's fine it can wait.